htdocs/core/actions_sendmails.inc.php

```diff
@@ -324,7 +324,7 @@
 			// <img alt="" src="'.$urlwithroot.'viewimage.php?modulepart=medias&amp;entity=1&amp;file=image/ldestailleur_166x166.jpg" style="height:166px; width:166px" />
 			$message = preg_replace('/(<img.*src=")[^\"]*viewimage\.php([^\"]*)modulepart=medias([^\"]*)file=([^\"]*)("[^\/]*\/>)/', '\1'.$urlwithroot.'/viewimage.php\2modulepart=medias\3file=\4\5', $message);
 
-			$sendtobcc = GETPOST('sendtoccc');
+			$sendtobcc = GETPOST('sendtocc');
```

Comment:
The bcc is the Blind Carbon Copy that is different of the Carbon Copy.
cc is store into "sendtocc"
bcc is stored into input  "sendtobcc"
So old code seems correct.

Reply:
Thank you for correcting my misread. Do I need to revert the pull-request or have you already done so?

Merci d'avoir corrigé ma mauvaise lecture. Dois-je annuler la pull-request ou l'avez-vous déjà fait ?

Reply:
> Thank you for correcting my misread. Do I need to revert the pull-request or have you already done so?
> 
> Merci d'avoir corrigé ma mauvaise lecture. Dois-je annuler la pull-request ou l'avez-vous déjà fait ?
> J'apprends encore le protocole des pull-requests.

Nothing to do, the PR was just no validated. You can cancel it yourself, or it will be done later if you agree the answer...